pyvex/__init__.py

```diff
@@ -40,7 +40,7 @@ def _find_c_lib():
     dir(lib)
     return lib
 
-pvc = _find_c_lib()
+pvc = _find_c_lib() # type: Any # This should be properly typed, but this seems non trivial
```

Comment:
Probably leave this un-annotated if not planning to properly type it out?

Reply:
https://github.com/angr/pyvex/blob/master/pyvex_c/pyvex.def ;)